### DISCUSSION TITLE
is it possible to name a deployment?

### DISCUSSION THREAD

**Discussion by its-danny** [vercel/vercel#4263] [General] [answered]
I have an API hosted on heroku, with a frontend app hosted on vercel. I'd like to be able to link these together easily by each review/preview app having predictable names. On the Heroku side, it can be set to `prefix-{prNumber}`. Is it possible to do the same on vercel?

**Comment by wiiiimm** [comment, marked as answer]
Vercel does not provide ways for you to do this. The closest feature we have is [Branch Domains](https://vercel.com/blog/branch-domains).
